src/router/router.js

```diff
@@ -46,6 +49,21 @@ export default new Router({
         },
       ],
     },
+    {
+      path: '/:username/profile',
+      name: 'profile',
+      component: Profile,
+    },
+    {
+      path: '/:username/my-events',
+      name: 'my-events',
```

Comment:
For all of the router links above you can do :to={name: my-events} for each route.